distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -26,7 +26,7 @@ prepare_visualization y max_rows=1000 = Helpers.recover_errors <|
             dataframe = x.take (First max_rows)
             all_rows_count = x.row_count
             included_rows = dataframe.row_count
-            index = Dataframe_Column.from_vector "#" (Vector.new included_rows i->i)
+            index = Dataframe_Column.from_vector "#" (Vector.new included_rows i->(i+1))
```

Comment:
I appreciate the more human-friendliness of this, and as we discussed, I'm not opposed.

But I feel we may get some hiccups due to this being inconsistent with `table.rows`. Could you please add a short-note to `Table.rows` docstring:

```
Differently from the table visualization where the row number is starting from 1, the returned vector is 0-indexed as all Enso collections.
```

(or something similar along these lines)

Probably also a similar thing would be good for `column.at`:
```
```
Differently from the table visualization where the row number is starting from 1, the this method indexes column rows starting from 0.
```

Reply:
Reverted to 0 numbering...